src/idm/user.js

```diff
@@ -0,0 +1,122 @@
+import axios from 'axios';
+import { errorHandler } from '../utils';
+import parseUrl from 'url-parse';
+
+const idm = (base, token) => {
+  base = `${base}`;
+  return ({
+    /**
+    * @summary Get the user information
+    * @name getUserInfo
+    * @public
+    * @function
+    * @memberof agile.idm.user
+    * @fulfil {Object} userInfo - object with user information
+    * @returns {Promise}
+    * @example
+    * agile.idm.user.getuserInfo().then(function(info) {
+    *  console.log(info);
+    * });
+    **/
+    getUserInfo: () => axios({
+      method: 'GET',
+      url: `${base}/oauth2/api/userinfo`,
+      headers: {"Authorization":`bearer ${token}`}
+    })
+    .then(res => (res.data))
+    .catch(errorHandler),
+    /**
+    * @summary Show information for a particular user
+    * @name getuser
+    * @public
+    * @function
+    * @memberof agile.idm.user
+    * @param {String} user_name user name
+    * @param {String} auth_type authentication type
+    * @fulfil {Object} user found
+    * @returns {Promise}
+    * @example
+    * agile.idm.user.getuser("alice","agile-local").then(function(user) {
+    *   console.log(user);
+    * });
+    **/
+    getUser: (user_name, auth_type) => {
+      var parsed = parseUrl(`${base}/api/v1/user/`);
+      parsed.set("query",{'auth_type': auth_type, 'user_name': user_name});
```

Comment:
Nit: Can't you just pass the params as an object to the `params` key on the axios instance?